File: tests/fixtures/realistic/express/services/accounts.ts
import { PrismaClient } from "@prisma/client";

const prisma = new PrismaClient();

export async function createAccount(ownerId: string, name: string) {
  return prisma.account.create({
    data: { ownerId, name },
  });
}

export async function updateAccount(accountId: string, data: Record<string, unknown>) {
  return prisma.account.update({
    where: { id: accountId },
    data,
  });
}

export async function deleteAccount(accountId: string) {
  return prisma.account.delete({
    where: { id: accountId },
  });
}

File: tests/fixtures/realistic/fastapi/app/services/accounts.py
from sqlalchemy import delete, update
from sqlalchemy.orm import Session

from app.models import Account


session = Session()


def create_account(owner_id: str):
    account = Account(name="service", owner_id=owner_id)
    session.add(account)
    session.commit()
    return account


def update_account(account_id: str, data):
    session.execute(update(Account))
    return {"id": account_id, **data}


def delete_account(account_id: str):
    session.execute(delete(Account))
    return {"deleted": account_id}
